packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -425,7 +425,9 @@ export function useSelectableCollection(options: AriaSelectableCollectionOptions
 
     // If the focused key becomes null (e.g. the last item is deleted), focus the whole collection.
     if (manager.isFocused && manager.focusedKey == null && lastFocusedKey.current != null) {
-      focusSafely(ref.current);
+      if (!shouldUseVirtualFocus) {
```

Comment:
merge into above if condition?

Reply:
ah whoops, forgot to do so after removing the extra focus key setting code, thanks